packages/odyssey/src/scss/abstracts/_mixins.scss

```diff
@@ -26,6 +26,8 @@
   box-shadow: 0 0 0 $radius $outline-cv;
 }
 
+/* stylelint-disable no-descending-specificity */
+
```

Comment:
:sob: is there no way to refactor to avoid this stylelint escape? perhaps just changing the order of the selectors themselves?

Reply:
Because each of these apply to subsequent states, changing the order as recommended by stylelint then gives a violation to change it back. 😞

We have to do this in a few places where form element styling is contingent on multiple states (disabled + invalid, for ex). I can hunt around to see if the SL config lets us be more/less specific in these instances.